configure.ac

```diff
@@ -11,8 +11,8 @@ AC_PREREQ([2.63])
 define([scm_r], esyscmd([sh -c "git rev-parse --short=7 HEAD"]))
```

Comment:
First we have to check if git is present  and does not return and error. We have but open for this. Please address this issue as well.